.github/workflows/ci.yml

```diff
@@ -127,6 +126,70 @@ jobs:
           path: openapi.yaml
           retention-days: 1
 
+  e2etests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        browser:
+          - chromium
+          - firefox
```

Comment:
In the docs, it says that also `webkit` is an option, is there a reason why we don't include it in CI?

Reply:
added and CI seems to pass